docker/root/etc/cont-init.d/20-config

```diff
@@ -1,15 +1,17 @@
 #!/usr/bin/with-contenv bash
 
-# make folders
-mkdir -p /database
-
 # copy config
 if [ ! -f "/config/settings.json" ]; then
   cp -a /defaults/settings.json /config/settings.json
 fi
 
+# check if db exists
+if [ ! -f "/database/filebrowser.db" ]; then
+  touch "/database/filebrowser.db"
+fi
+
 # permissions
-chown abc:abc \
```

Comment:
abc user equls to the provied pid and guid https://github.com/linuxserver/docker-baseimage-alpine/blob/master/root/etc/s6-overlay/s6-rc.d/init-adduser/run#L6-L8

Reply:
And yet it fails at that stage it seems. Running the following:
```bash
mkdir -p /tmp/fb/root /tmp/fb/db /tmp/fb/config
chown -R 99:100 /tmp/fb

docker run \
    -v /tmp/fb/root:/srv \
    -v /tmp/fb/db:/database \
    -v /tmp/fb/config:/config \
    -e PUID=99 \
    -e PGID=100 \
    -p 8809:80 \
    filebrowser/filebrowser:s6
```
Yields the following permissions on the files/folders
```bash
ls lha /tmp/fb
total 0
drwxrwxrwx 2     99   100 60 Dec 29 12:57 config
drwxrwxrwx 2    911  1001 60 Dec 29 12:57 db
drwxrwxrwx 2    911  1001 40 Dec 29 12:56 root

ls lha /tmp/fb/config
total 4.0K
-rw-r--r-- 1     99  1001 129 Nov  2 22:57 settings.json

ls lha /tmp/fb/db
total 36K
-rw------- 1     99   100 64K Dec 29 12:57 filebrowser.db

ls lha /tmp/fb/root
total 0
```